src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/plantation/modules/generic/BoneMealedPlantModule.java

```diff
@@ -182,6 +177,13 @@ public List<ItemStack> getRequiredItemsForOperation()
         return getValidBonemeal().stream().map(ItemStack::new).toList();
     }
 
+    @Nullable
+    @Override
+    protected Integer getMaxWorkingPositions()
```

Comment:
You add for a few and remove for a few, difficult to understand what that changes.